.rubocop_todo.yml

```diff
@@ -9,20 +9,26 @@
 # Offense count: 1
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Max: 42
+  Max: 28
 
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
   Max: 227
 
+# Offense count: 8
+# Configuration parameters: Max.
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/features/availability_spec.rb'
+
```

Comment:
I just added a exclusion on ExampleLength for all features. Cool with that?

Reply:
👍